script.js

```diff
@@ -1 +1,66 @@
-// 다들 화이팅!! ٩( *˙0˙*)۶
\ No newline at end of file
+// 다들 화이팅!! ٩( *˙0˙*)۶
+const todoInput = document.querySelector('#todoInput'); //입력창
+const addBtn = document.querySelector('#addBtn'); //+ 버튼
+const todoList = document.querySelector('#todoList'); //TO DO
+const doneList = document.querySelector('#doneList'); //DONE
+const todoTitle = document.querySelector('.todo-box .list-title'); // TO DO 제목
+const doneTitle = document.querySelector('.done-box .list-title'); // DONE 제목
+
+function EnterKeyDown() {
+  //엔터키로 추가
+  if (window.event.keyCode === 13 && todoInput.value !== '') {
+    addToDo();
+  }
+}
+
+addBtn.addEventListener('click', () => {
+  //+ 버튼으로 추가
+  if (todoInput.value !== '') addToDo();
+});
+
+function addToDo() {
+  let todoContents = todoInput.value;
+
+  const li = document.createElement('li');
+  const span = document.createElement('span');
+  const btn = document.createElement('button');
+
+  li.appendChild(span); //appendChild() : 선택한 요소 안에 자식요소를 추가
+  li.appendChild(btn);
+
+  span.textContent = todoContents; //textContent : 노드 내의 모든 텍스트 추출
+  btn.textContent = '✖️';
+
+  todoList.appendChild(li);
+
+  todoInput.value = ' ';
```

Comment:
이렇게 되면 할 일 추가 후 인풋 필드에 공백이 남아 있게 되니 아래처럼 수정해야 할 것 같습니다!
```suggestion
  todoInput.value = '';
```